contacts/index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+
+<html>
+
+    <head>
+        <title>Chat Room</title>
+    </head>
+
+    <body>
+        
+<!-- ------------------------------------------------------------------------------------------------------ -->
+
+        <header>
+            <h1>Chat Room</h1>
+        </header>
+
+<!-- ------------------------------------------------------------------------------------------------------ -->
+
+        <nav>
+            <ul>
+                <li><a href="D:\code\mini_project\index.html">Home</a></li>
+                <li><a href="index.html">Contacts</a></li>
+            </ul>
+        </nav>
+        
+<!-- ------------------------------------------------------------------------------------------------------ -->
+
+        <main>
+
+            <h2>Contacts</h2>
+
+            <ul>
+
+                <li>
+                    <div> 
+                        <button>+</button>
+                        <label>0</label>
+                        <button type="button">-</button>
```

Comment:
type="button" is not required, as type=button is the default value of the button tag. Update all buttons on this page, please.